x-pack/plugins/ml/public/application/jobs/jobs_list/components/reset_job_modal/reset_job_modal.tsx

```diff
@@ -124,6 +126,14 @@ export const ResetJobModal: FC<Props> = ({ setShowFunction, unsetShowFunction, r
                 jobsCount: jobIds.length,
               }}
             />
+            <EuiSpacer />
+            <EuiSwitch
+              label={i18n.translate('xpack.ml.jobsList.resetJobModal.deleteUserAnnotations', {
+                defaultMessage: 'Also delete annotations.',
```

Comment:
Looking at the text we have in the modal when deleting DFA jobs, maybe this could just be `Delete annotations` ?

Reply:
Text updated in https://github.com/elastic/kibana/pull/147537/commits/256bd0eda575351b54568ab6ad6b9a973010ccd8
Added functional tests in https://github.com/elastic/kibana/pull/147537/commits/9366e4c219aad7606a75352308937fcdd1d88782